src/renderers/shaders/ShaderChunk/bsdfs.glsl

```diff
@@ -0,0 +1,152 @@
+//#define ENERGY_PRESERVING_MONOCHROME
+
+float calcLightAttenuation( float lightDistance, float cutoffDistance, float decayExponent ) {
+
+	if ( decayExponent > 0.0 ) {
+
+	  return pow( saturate( -lightDistance / cutoffDistance + 1.0 ), decayExponent );
+
+	}
+
+	return 1.0;
+
+}
+
+
+void BRDF_Lambert( const in IncidentLight incidentLight, const in GeometricContext geometryContext, const in vec3 diffuseColor, inout ReflectedLight reflectedLight ) {
+
+	float lambertianReflectance = saturate( dot( geometryContext.normal, incidentLight.direction ) );
+
+	#if defined( ENERGY_PRESERVING_MONOCHROME ) || defined( ENERGY_PRESERVING_RGB )
+
+		lambertianReflectance *= RECIPROCAL_PI;
```

Comment:
@WestLangley wrote:
> The equations that are implemented in three.js are detailed in Background: Physics and Math of Shading.

Thank you, I will update the PR to use this terminology exactly.  This will help.

@WestLangley wrote:
> The reflectance equation has a factor of pi in front

Actually, the reflectance equation, i.e. equation (1), in the above doc does not have a PI factor in it.  I think somehow we got confused in thinking it did and then optimized out the divide by PI erroneously in the Lambert BRDF / render equation.  I am pretty sure it should go back in -- we should be dividing by PI.